docs/charts/polar.md

```diff
@@ -94,7 +94,7 @@ These are the customisation options specific to Polar Area charts. These options
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | -----------
-| `startAngle` | `number` | `0` | Starting angle to draw arcs for the first item in a dataset. In degrees, 0 is at top.
+| `startAngle` | `number` | `-0.5 * Math.PI` | Starting angle to draw arcs for the first item in a dataset.
```

Comment:
The mention of where the top is has been removed, which I think was useful. Maybe add back "In radians, 0 is at the top"

Reply:
I just reverted this change, because its not changed to "top" anymore, but stays "right" as it was before.